backend/yearbook/src/main/java/com/obsidi/yearbook/jpa/Profile.java

```diff
@@ -0,0 +1,226 @@
+package com.obsidi.yearbook.jpa;
+
+	import java.io.Serializable;
+
+
+
+	import com.fasterxml.jackson.annotation.JsonIgnore;
+	import com.fasterxml.jackson.annotation.JsonProperty;
+	import com.fasterxml.jackson.annotation.JsonProperty.Access;
+
+	import jakarta.persistence.Column;
+	import jakarta.persistence.Entity;
+	import jakarta.persistence.GeneratedValue;
+	import jakarta.persistence.GenerationType;
+	import jakarta.persistence.Id;
+	import jakarta.persistence.JoinColumn;
+	import jakarta.persistence.OneToOne;
+	import jakarta.persistence.Table;
+
+
+
+	@Entity
+	@Table(name="\"Profile\"")
+	public class Profile implements Serializable {
+
+		private static final long serialVersionUID = 1L;
+		
+		@Id
```

Comment:
Hi Amy,

The table's Ids are not of type integer, but rather UUID. These are the annotations required:
``` java
  @Id
  @ColumnDefault("gen_random_uuid()")
  @Column(name = "\"profileId\"", nullable = false)
  private UUID profileId;
```
Could you please update them, we'll have to update for the user table as well.
Thanks